db/migrate/20250225000050_remove_contents_from_issue_contents.rb

```diff
@@ -0,0 +1,95 @@
+# For auto load
+FullTextSearch::Migration
+
+class RemoveContentsFromIssueContents < ActiveRecord::Migration[5.2]
```

Comment:
What version of Redmine did you use to generate this migration file?

It seems that `[5.2]` is too old.

Reply:
fix: [2d10410](https://github.com/clear-code/redmine_full_text_search/pull/165/commits/2d10410d417a267fcc4713781e8e51f0fb5cf81a) Thanks you so much. I thought the minimum Rails version in supported Redmine was 5.2 but it is 6.1 now. So I updated it.
- ref: https://www.redmine.org/projects/redmine/wiki/Download
- ref: https://github.com/redmine/redmine/blob/5.0-stable/Gemfile#L6